pgmpy/models/FactorGraph.py

```diff
@@ -391,3 +391,39 @@ def get_partition_function(self):
             raise ValueError('DiscreteFactor for all the random variables not defined.')
 
         return np.sum(factor.values)
+
+    def copy(self):
+        """
+        Returns a copy of the model.
+
+        Returns
+        -------
+        FactorGraph : Copy of FactorGraph
+
+        Examples
+        --------
+        >>> from pgmpy.models import FactorGraph
```

Comment:
@lohani2280 Import order should be (PEP 8): 
1. standard library imports
2. related third party imports
3. local application/library specific imports